extensions/interactions/interaction_specs.json

```diff
@@ -972,15 +972,15 @@
     "is_terminal": false,
     "default_outcome_heading": null,
     "rule_descriptions": {
+      "IsExactlyEqualTo": "exactly matches {{f|Fraction}}",
+      "IsEquivalentTo": "is equivalent to {{f|Fraction}}",
+      "IsEquivalentToAndInSimplestForm": "is equivalent to {{f|Fraction}} and in simplest form",
       "HasNoFractionalPart": "has no fractional part",
       "HasFractionalPartExactlyEqualTo": "has fractional part exactly equal to {{f|Fraction}}",
```

Comment:
Why is the ordering changed in this file?

Reply:
I initially changed this to change the ordering of the rules displayed in the editor so that more common rules like 'Equals' are displayed first, but as @seanlip said above, this may not work for all browsers as it is a dict. So, I'll keep this new ordering and enforce later on, that 'Equals' rule is displayed first.

Reply:
Done, reverted these.